liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -646,4 +650,7 @@ private void makeParentDirectory(File file) {
         }
     }
 
+    public static boolean isConfigCopied() {
```

Comment:
Does this method really need to be static?

Reply:
Good question.   I think it might have to be an esoteric use case before this would matter, like having multiple executions deploying to more than one server in one mvn invocation, in which case a single, static flag might not map well.
(Interestingly, the ServerConfigDocument cache will invalidate its entry and create a new one if the server location change, so doesn't have such an issue).

But it seems like we should err on the side of NOT building the ServerConfigDocument cache , since we can handle `getAppsDirectory()` returning the default ("dropins") better than we can handle building the cache instance when we shouldn't.

So I think this argues for using an instance variable rather than static.